command/cp.go

```diff
@@ -836,3 +860,33 @@ func givenCommand(c *cli.Context) string {
 
 	return cmd
 }
+
+func createExcludeUrls(ctx context.Context, excludes []string, client storage.Storage, srcurls ...*url.URL) ([]*url.URL, error) {
```

Comment:
I believe it would be better if this function (and `strutil.IsURLExcluded` ) had its own file, like `expand.go` for `expandSource`.

Reply:
Could you also add tests for it?

Reply:
moved this function to `command/exclude.go` and wrote unit tests.